android/jni/tun2socks/tun2socks.c

```diff
@@ -128,7 +129,8 @@ struct {
     int set_signal;
     // ==== PSIPHON ====
     // ==== UPROXY ====
-    char *dns_server_address;
+    char *dns_resolver_address;
```

Comment:
Definitely worth adding some comments here. Am I right to say:
* `dns_resolver_address`: address of the DNS server to which all DNS requests will be forwarded
* `udp_relay_address`: address to which all SOCKS UDP requests will be sent (NOTE: this implementation **does not make** UDP ASSOCIATE requests)